app/main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self, name: str, health: int = 100, hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        name, health, hidden = self.name, self.health, self.hidden
```

Comment:
does it needed variables? You can make the same options without them

Reply:
whith out them flake8 will be crash

Reply:
ok, i`m fixed it